anuvaad-etl/anuvaad-extractor/file_translator/etl-file-translator/services/docx_transform.py

```diff
@@ -246,15 +255,16 @@ def add_run_as_para_json(self, para):
         try:
             self.para_count += 1
 
-            for idr, run in enumerate(common_obj.get_runs(para, para_obj=True)):
-                self.run_count += 1
-                json_run = self.generate_para_struct_json_for_run(run=run, file_id=self.file_id,
-                                                                  para_idx=str(self.sequence_para_index),
-                                                                  run_idx=str(idr))
-                words_no = common_obj.word_count(json_run.get('text'))
-                self.word_count += words_no
-                json_run['children'].append(copy.deepcopy(json_run))
-                self.page_list[self.page_number - 1]['text_blocks'].append(json_run)
+            for runs in common_obj.get_runs(para, para_obj=True):
+                for idr, run in runs:
```

Comment:
have you tested this method? 
for idr, run in runs:
this line will through error.

Reply:
It shouldn't give an error because get_runs returns a list of lists from which we are extracting individual lists as runs. Then I'm using 'for idr, run in runs:' to further enumerate the runs within each nested list of runs. I will test it again anyway.

Reply:
the correct code is:
for  run in runs:
or
for idr, run in enumerate(runs):

Reply:
Oh sorry, I meant to use enumerate here. Didn't catch that, thanks!